src/fastserve/models/huggingface.py

```diff
@@ -0,0 +1,79 @@
+import logging
+import os
+from typing import Any, List
+
+from pydantic import BaseModel
+from transformers import AutoModel, AutoTokenizer
+
+from fastserve.core import FastServe
+
+# Configure logger
+logger = logging.getLogger(__name__)
+
+
+class PromptRequest(BaseModel):
+    prompt: str = "Write a python function to resize image to 224x224"
+    temperature: float = 0.8
+    top_p: float = 1.0
+    max_tokens: int = 200
+    stop: List[str] = []
+
+
+class ServeHuggingface(FastServe):
```

Comment:
can we rename it to `ServeHuggingFace` please?

Reply:
Sure! 👍🏼